account-decoder/src/lib.rs

```diff
@@ -44,6 +49,8 @@ pub enum UiAccountEncoding {
     Base58,
     Base64,
     JsonParsed,
+    #[serde(rename = "base64+zstd")]
```

Comment:
WDYT about the encoding name `"base64+zstd"`?

Reply:
I like it!